extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -447,6 +479,16 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         isTrafficEligible: () => true,
         branches: Object.values(PTT_EXP_BRANCHES),
       },
+      {
+        experimentId: IDLE_CWV_EXP,
+        isTrafficEligible: () => () => {
```

Comment:
is this intended to be a nested function?

Reply:
nope :-). updated